### PR TITLE
Type checking with prop types

### PR DESCRIPTION
**Pull request by sucetin**

### CODE REVIEW

File: src/components/InputWithLabel.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
 import React, { useRef, useEffect } from "react";
 import styles from "./InputWithLabel.module.css";
+import PropTypes from "prop-types";
 
 function InputWithLabel({ todoTitle, handleTitleChange, children }) {
 	const inputRef = useRef();
 	useEffect(() => {
 		inputRef.current.focus();
 	});
+	InputWithLabel.propTypes = {
```
Comment (davidrapoport) [inline src/components/InputWithLabel.js:10]: It looks like the official recommendation is to do this outside of the function (probably to keep things neater)

https://reactjs.org/docs/typechecking-with-proptypes.html#function-components

File: src/components/TodoList.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import TodoListItem from "./TodoListItem";
+import PropTypes from "prop-types";
+
+function TodoList({ todoList, onRemoveTodo }) {
```
Comment (davidrapoport) [inline src/components/TodoList.js:5]: use `git mv` to keep the file history

**Review by davidrapoport — APPROVED** [review]
Good job!

File: src/components/InputWithLabel.js (hunk 1 above)
Comment (sucetin) [reply src/components/InputWithLabel.js:10]: Let's talk about it on Tuesday!

**Pull request merged**